pkgs/by-name/ll/lldap-cli/package.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, resholve
+
+, bash
+, coreutils
+, curl
+, gnugrep
+, gnused
+, jq
+, lldap
+, util-linux
+}:
+
+resholve.mkDerivation rec {
+  pname = "lldap-cli";
+  version = "0-unstable-2024-05-15";
+
+  src = fetchFromGitHub {
+    owner = "ibizaman";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21